core/src/main/java/io/github/tcdl/msb/api/MsbContextBuilder.java

```diff
@@ -63,7 +78,17 @@ public MsbContextBuilder withDefaultChannelMonitorAgent(boolean withDefaultChann
         this.withDefaultChannelMonitorAgent = withDefaultChannelMonitorAgent;
         return this;
     }
-    
+
+    /**
+     * Specifies payload mapper to serialize/deserialize message payload
+     * @param payloadMapper if not provided default mapper will be used
```

Comment:
mapper seems to be not so obvious name(try to google for "java mapper"). Json ObjectMapper or  ObjectMapper will be better